### PR TITLE
Add service view template

### PR DESCRIPTION
**Pull request by YitongFeng**
Add service view template, now it is an empty table waiting for data.

image

### CODE REVIEW

File: webportal/src/app/cluster-view/services/services.component.js [hunk 1]
```diff
@@ -22,4 +22,51 @@ const servicesComponent = require('./services.component.ejs');
...
 
-$('#content-wrapper').html("<div>TODO: List services on each node here.</div>");
\ No newline at end of file
+const breadcrumbComponent = require('../../job/breadcrumb/breadcrumb.component.ejs');
+const loadingComponent = require('../../job/loading/loading.component.ejs');
+const serviceTableComponent = require('./service-table.component.ejs');
+const serviceViewComponent = require('./services.component.ejs');
+const loading = require('../../job/loading/loading.component');
+//const webportalConfig = require('../../../config/webportal.config.json');
+const userAuth = require('../../user/user-auth/user-auth.component');
```
Comment (abuccts) [inline webportal/src/app/cluster-view/services/services.component.js]: move `require` block to the beginning of js file

File: webportal/src/app/cluster-view/services/services.component.js [hunk 2]
```diff
@@ -22,4 +22,51 @@ const servicesComponent = require('./services.component.ejs');
 $("#sidebar-menu--cluster-view").addClass("active");
 $("#sidebar-menu--cluster-view--services").addClass("active");
```
Comment (abuccts) [inline webportal/src/app/cluster-view/services/services.component.js]: move these before `$(document).ready`, and use single quotes in js please

File: webportal/src/app/cluster-view/services/services.component.js (hunk 1 above)
Comment (YitongFeng) [reply webportal/src/app/cluster-view/services/services.component.js]: Solved

File: webportal/src/app/cluster-view/services/services.component.js (hunk 2 above)
Comment (YitongFeng) [reply webportal/src/app/cluster-view/services/services.component.js]: Solved

**Pull request merged**